packages/terra-site/webpack.config.js

```diff
@@ -6,6 +6,7 @@ const path = require('path');
 const autoprefixer = require('autoprefixer');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const I18nAggregatorPlugin = require('./node_modules/terra-i18n/lib/I18nAggregatorPlugin');
```

Comment:
Yeah I agree there's a weird mix of development dependencies and runtime dependencies going on here.  We can discuss it tomorrow in the war room.